### PR TITLE
Added tags to model card

### PR DESCRIPTION
**Pull request by merveenoyan**
The code downloads the model, looks at config and adds necessary tags. 
For instance, this is what it creates for English Stanza model:
```

---
tags:
- stanza
- token-classification
- ner 
- pos 
- sentiment-analysis 

library_name: stanza
language:
- en
license: apache-2.0
---
# Stanza model for English (en)
Stanza is a collection of accurate and efficient tools for the linguistic analysis of many human languages. Starting from raw text to syntactic analysis and entity recognition, Stanza brings state-of-the-art NLP models to languages of your choosing.
Find more about it in [our website](https://stanfordnlp.github.io/stanza) and our [GitHub repository](https://github.com/stanfordnlp/stanza).

This card and repo were automatically prepared with `hugging_stanza.py` in the `stanfordnlp/huggingface-models` repo

```

cc: osanseviero @AngledLuffa

### CODE REVIEW

File: hugging_stanza.py [hunk 1]
```diff
@@ -48,6 +57,23 @@ def write_model_card(repo_local_path, model):
...
+    input_dir = args.input_dir
+
+    stanza.download(model_dir = input_dir, lang = lang)
+    model = Pipeline(model_dir= input_dir, lang=lang)
+    config_keys = model.config.keys()
+    tags = []
+    if "ner_model_path" in config_keys:
+        tags.append("ner")
+    if "pos_model_path" in config_keys:
+        tags.append("pos")
```
Comment (osanseviero) [inline hugging_stanza.py:71]: These should be the full name based on https://github.com/huggingface/huggingface_hub/blob/main/widgets/src/lib/interfaces/Types.ts

`named-entity-recognition` and `part-of-speech-tagging`

**Review by osanseviero — COMMENTED** [review]
Do you have an example repo created with this script?

File: hugging_stanza.py (hunk 1 above)
Comment (merveenoyan) [reply hugging_stanza.py]: fixed 🤗

**Comment by osanseviero** [comment]
Instead of loading the model, couldn't we just look if models/sentiment, models/pos, and models/ner directories exist under `dst` after doing `copytree`? That would be easier and faster

**Comment by AngledLuffa** [comment]
Yes, that is what I was trying to say.  There's also depparse and conparse for various languages if those tags are relevant.

**Comment by merveenoyan** [comment]
I'll submit another PR, thanks a lot for the pointers!

**Pull request closed without merging**